### PR TITLE
Customer request service

### PR DESCRIPTION
**Pull request by Gabetn**

### CODE REVIEW

File: src/main/js/components/App.js [hunk 1]
```diff
@@ -212,6 +213,32 @@ export class App extends React.Component {
         });
     }
 
+    updateDiningSession(diningSession, updatedDiningSession, option, string) {
+
+        updatedDiningSession['tableNumber'] = diningSession.tableNumber;
+        updatedDiningSession['diningSessionStatus'] = diningSession.diningSessionStatus;
+        updatedDiningSession['serviceRequestStatus'] = diningSession.serviceRequestStatus;
+        updatedDiningSession['billRequestStatus'] = diningSession.billRequestStatus;
+        updatedDiningSession['tableAssignmentStatus'] = diningSession.tableAssignmentStatus;
```
Comment (Gabetn) [inline src/main/js/components/App.js]: No need to create the entire object, can simply modify the attribute based on your switch case. See "PATCH" in onUpdate

File: src/main/js/components/App.js [hunk 2]
```diff
@@ -212,6 +213,32 @@ export class App extends React.Component {
         });
     }
 
+    updateDiningSession(diningSession, updatedDiningSession, option, string) {
+
+        updatedDiningSession['tableNumber'] = diningSession.tableNumber;
```
Comment (Gabetn) [inline src/main/js/components/App.js:218]: Can keep tablenumber

File: src/main/js/components/App.js [hunk 3]
```diff
@@ -566,12 +578,29 @@ export class App extends React.Component {
                                                    orders={this.state.orders}
                                                    orderLinks={this.state.orderLinks}
                                                    orderAttributes={this.state.orderAttributes}
-                                                   menuItems={this.props.menuItems}
```
Comment (Gabetn) [inline src/main/js/components/App.js:569]: This seems to have been removed by accident Same with line 570

**Review by EBruchet — APPROVED** [review]
**Issues Fixed:**
- There were unnecessary setting of certain attributes for a dining session, this was removed.

**Pull request merged**